Source/WTF/wtf/ContinuousArenaMalloc.cpp

```diff
@@ -161,12 +161,12 @@ void* ContinuousArenaMalloc::extentAlloc(extent_hooks_t *extent_hooks,
     if (new_addr != NULL || size == 0) {
         ret = NULL;
     } else {
-        s_Current = __builtin_align_up(s_Current, alignment);
+        char *start = __builtin_align_up(s_Current, alignment);
```

Comment:
It's unlikely to make any difference but I wondered if this might better be `void *`?

Reply:
I wrote this a while ago so I'm going to have to try to remember the reasoning here. I think I picked `char *` here because that's what `s_Current` is, but also because I'd have to cast it back to `char *` (or similar) for the arithmetic on line 185 anyway.

I think I did use `void *` (and a cast) [in a later patch that reworks](https://github.com/capablevms/webkit/pull/5/commits/f4a918461a325254e4a77ae5e6d9ec2d70ad766b#diff-5351ae11e716c74cc38c0f110f21479865cd71969765ba290c2236718fb8dea4R189). I don't remember why it made sense at that point, though.

Reply:
OK, let's leave this as-is.